### PR TITLE
Feature/multiple batch clients

### PR DESCRIPTION
**Pull request by darthkurak**

### CODE REVIEW

File: src/Allegro.CosmosDb.BatchUtilities/Allegro.CosmosDb.BatchUtilities/Allegro.CosmosDb.BatchUtilities.csproj [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <PackageId>Allegro.CosmosDb.BatchUtilities</PackageId>
     <PackageDescription>Utilities for performing batch operations in Azure CosmosDb, such as rate limiting and autoscaling.</PackageDescription>
-    <Version>1.0.3</Version>
+    <Version>2.0.16</Version>
```
Comment (starkpl) [inline src/Allegro.CosmosDb.BatchUtilities/Allegro.CosmosDb.BatchUtilities/Allegro.CosmosDb.BatchUtilities.csproj]: why so high?

File: src/Allegro.CosmosDb.BatchUtilities/CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres
 to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.0.0] - 2022-09-01
```
Comment (starkpl) [inline src/Allegro.CosmosDb.BatchUtilities/CHANGELOG.md]: Version does not match with csproj

File: src/Allegro.CosmosDb.BatchUtilities/Allegro.CosmosDb.BatchUtilities/Allegro.CosmosDb.BatchUtilities.csproj (hunk 1 above)
Comment (darthkurak) [reply src/Allegro.CosmosDb.BatchUtilities/Allegro.CosmosDb.BatchUtilities/Allegro.CosmosDb.BatchUtilities.csproj]: for my testing, before completing, i will set 2.0.0

File: src/Allegro.CosmosDb.BatchUtilities/CHANGELOG.md (hunk 2 above)
Comment (darthkurak) [reply src/Allegro.CosmosDb.BatchUtilities/CHANGELOG.md]: Is higher for my testing, before completing, i will set 2.0.0

**Review by starkpl — APPROVED** [review]

**Review by ppiwow-apay — APPROVED** [review]

**Pull request merged**